src/Integration.UnitTests/Connection/ConnectControllerTests.cs

```diff
@@ -93,6 +96,7 @@ public void ConnectionController_ConnectCommand_Status()
         {
             // Arrange
             var testSubject = new ConnectionController(this.host);
+            this.activeSolutionTracker.SimulateActiveSolutionChanged(true);
```

Comment:
Are there any tests that check the connection status when the solution is not loaded?

Reply:
I have simply added tests for the new case so I can't tell you whether the other was already covered or not.

Reply:
I meant tests simulating the event with _IsSolutionOpen = false_ , but on looking again I see you've added a default parameter to _SimulateActiveSolutionChanged_ so I guess there are tests that cover it.